storage/storage-client/src/lib.rs

```diff
@@ -54,6 +55,63 @@ use storage_proto::{
 };
 use tokio::runtime::Runtime;
 
+pub struct StorageServiceClientSimple {
+    network_client: Mutex<NetworkClient>,
+}
+
+impl StorageServiceClientSimple {
+    pub fn new(server_address: &SocketAddr) -> Self {
+        Self {
+            network_client: Mutex::new(NetworkClient::new(*server_address)),
+        }
+    }
+
+    fn request(
+        &self,
+        input: storage_interface::StorageMsg,
+    ) -> Result<Vec<u8>, storage_interface::Error> {
+        let input_message = lcs::to_bytes(&input)?;
+        let mut client = self.network_client.lock().unwrap();
+        client.write(&input_message)?;
+        let result = client.read()?;
+        Ok(result)
+    }
+
+    pub fn get_account_state_with_proof_by_version(
```

Comment:
following my comment above, this then becomes just a specialization around the request function above... not a big issue one way or the other, but we don't need lcs::from_bytes(&response) multiple times.